packages/textfield/src/Textfield.ts

```diff
@@ -355,6 +357,7 @@ export class TextfieldBase extends ManageHelpText(
                 .value=${live(this.displayValue)}
                 @change=${this.handleChange}
                 @input=${this.handleInput}
+                @pointerdown=${this.handleInputElementPointerdown}
```

Comment:
This is interesting! Is this a deliberate to create a null event?

Reply:
OH my bad. This is surfaced up in NumberField

Reply:
Yeah, it's an annoyance of the class extension system and templating that this is required but the idea is definitely that by default Textfield does nothing for `pointer` down events and then in NumberField it does something specific. It implies the benefit of a alternative composition techniques within these render methods, but that's a huge alternative project with unclear benefits and risks.